assets/src/blocks/Spreadsheet/Spreadsheet.js

```diff
@@ -16,22 +16,38 @@ export class Spreadsheet extends Component {
   renderEdit() {
     const { __ } = wp.i18n;
 
-    const { color } = this.props;
+    const { attributes, setAttributes } = this.props;
+
+    const toAttribute = attributeName => value => {
+      setAttributes( { [ attributeName ]: value } );
+    };
 
     const colors = [
       { name: 'blue', color: '#C9E7FA' },
       { name: 'green', color: '#D0FAC9' },
       { name: 'grey', color: '#DCDCDC' },
     ];
 
+    const toCssVariable = ( variableName ) => ( value ) => {
+      setAttributes( {
+        css_variables: [
+          ...attributes.css_variables.filter( variable => variable.name !== variableName ),
+          {
+            name: variableName,
+            value: value
+          },
+        ]
+      } );
+    };
+
     return (
       <Fragment>
         <InspectorControls>
           <PanelBody title={__('Setting', 'p4ge')}>
             <ColorPaletteControl
               label={__('Table Color', 'p4ge')}
-              value={ color }
-              onChange={ this.props.onTableColorChange }
+              value={ attributes.color }
```

Comment:
Still need to make this read from the right variable, probably that will be easier to do if we make `CSS_VARIABLES_ATTRIBUTE` an object instead of an array. Then `value` would just be `attributes.css_variables['spreadsheet-row-background'`.